app/src/main/java/cm/aptoide/pt/home/FakeBundleDataSource.java

```diff
@@ -71,7 +71,8 @@
     appBundles.add(appBundle);
     appBundles.add(appBundle1);
     appBundles.add(appBundle2);
-    appBundles.add(new AdBundle("Highlighted", Collections.emptyList(), null, ""));
+    appBundles.add(
+        new AdBundle("Highlighted", new AdsTagWrapper(Collections.emptyList(), ""), null, ""));
```

Comment:
Should this be here?

Reply:
Just wanted to make sure that an AdBundle was included in the fake bundles list for the test, even tho it doesn't have any ads inside. The test is not verifying if ads are shown in the UI tho. I think we should make this test later, also fill this bundle with Ad objects.